setup.py

```diff
@@ -146,6 +146,7 @@
             'code-snippets = elyra.metadata.schemasproviders:CodeSnippetsSchemas',
             'component-registries = elyra.metadata.schemasproviders:ComponentRegistriesSchemas',
             'component-catalogs = elyra.metadata.schemasproviders:ComponentCatalogsSchemas',
+            'airflow-package-catalog-schema = elyra.pipeline.airflow.package_catalog_connector.airflow_package_schema_provider:AirflowPackageSchemasProvider',  # noqa: E501
```

Comment:
My main motivation was to to distinguish it from `airflow-provider-package-catalog` and `airflow-provider-package-catalog-schema` https://github.com/elyra-ai/elyra/blob/bebb5489b158252857f0206fd6d3da35a629f680/setup.py#L149. I'm okay with removing `package`  from all module names and class names if you believe this won't introduce ambiguity.

Reply:
I see.  I didn't realize the near duplicate PR #2438 - sigh.

what about something along the lines of:

AirflowBuiltinOperatorConnector for: https://airflow.apache.org/docs/apache-airflow/stable/_api/airflow/operators/index.html
and
AirflowProviderPackageConnector for: https://airflow.apache.org/docs/apache-airflow-providers/index.html

I would also recommend we place BOTH of these in the same directory within `airflow` leaving stanza entries like:

'airflow-operator-schema = elyra.pipeline.airflow.catalog_connectors.airflow_builtin_operators:AirflowBuiltinOperatorSchemas', 
'airflow-provider-package-schema = elyra.pipeline.airflow.catalog_connectors.airflow_provider_packages:AirflowProviderPackageSchemas'


'airflow-operator-connector = elyra.pipeline.airflow.catalog_connectors.airflow_builtin_operators:AirflowBuiltinOperatorConnector', 
'airflow-provider-package-connector = elyra.pipeline.airflow.catalog_connectors.airflow_provider_packages:AirflowProviderPackageConnector'


I suppose the separate directories are so that these can (eventually) be separately installable, etc. so the above recommendation is moot relative to that point.

I can see this isn't going to be worth pressing, just seems overly verbose to me - approving now.